.travis.yml

```diff
@@ -42,12 +48,19 @@ jobs:
         # Substitute build string
         - sed -i.bak "s/buildInfo = \"\"/buildInfo = \"$TRAVIS_BRANCH-$TRAVIS_COMMIT\"/" tapdance/logger.go
         - mkdir -p $GOPATH/src/github.com/Psiphon-Labs
+        # Get Specific Psiphon Core Library Release
         - git clone https://github.com/Psiphon-Labs/psiphon-tunnel-core.git $GOPATH/src/github.com/Psiphon-Labs/psiphon-tunnel-core
+        - (cd $GOPATH/src/github.com/Psiphon-Labs/psiphon-tunnel-core && git checkout $PSIPHON_CORE_VERSION -b build-refraction-networking)
```

Comment:
It appears that we previously would clone master branch, but now are cloning build-refraction-networking? Is there a reason for that?

Reply:
This pulls a commit based on an environment variable set in the travis ci online environment. I added this so that there is some consistency around what is being built against. It is just checking it out locally to a branch named `build-refraction-networking`.

This is actually a question that I have for the psiphon folks, if there are stable releases of this repo I would prefer to use those ( as we are for `Psiphon-Labs/psiphon-tunnel-core`) but these are configurable to specific commits / tags / releases using env vars in travis now.

This is the same for #23